pykob/config.py

```diff
@@ -706,9 +706,9 @@ def read_config():
         __option = "Code type"
         __key = __CODE_TYPE_KEY
         _code_type = (user_config.get(__CONFIG_SECTION, __key)).upper()
-        if _code_type == "AMERICAN":
+        if _code_type == "A" or _code_type == "AMERICAN":
             code_type = CodeType.american
-        elif _code_type == "INTERNATIONAL":
+        elif _code_type == "I" or _code_type == "INTERNATIONAL":
             code_type = CodeType.international
```

Comment:
Although the option processing supports 'A/a' and 'I/i', the configuration file always has 'AMERICAN' or 'INTERNATIONAL'. Are you suggesting that the configuration file also support having a value of 'A' or 'I'? I don't see the need for that. I prefer that the configuration file only have a single value. I don't really think people should be editing the configuration (soon to be 'preferences') file directly, but if they do they should be expected to use the defined property/option values.